doc/bcache.md

```diff
@@ -0,0 +1,79 @@
+# Bcache
+
+Bcache technology allows to speed up the access time (read and write) for old hard disks. The idea
```

Comment:
"speed up the access time" sounds strange. I suggest to either use ""speed up the access" or "reduce the access time".